install.sh

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2024 Greg Dumas gdumas@redhat.com
```

Comment:
This needs the full licensing, see front of bin/burden for it.

Reply:
Full license text added.